### ISSUE TITLE
Need I install web sdk for butterfly ?

### ISSUE THREAD

**Issue by goodforever** [yjbanov/butterfly#43]
I have installed flutter sdk , Need I install web sdk for building and running butterfly ?  Can  they(two sdk) be installed together?

**Comment by yjbanov** [comment]
Yes, you need to install the Web SDK from https://webdev.dartlang.org/tools/sdk#install to use this package. The two SDKs can coexist. Flutter bundles a stripped down Dart SDK under `bin/cache/dart-sdk`. Make sure you do not have it in your `PATH`. Instead put the Web SDK in your `PATH`. Otherwise when you call the `dart` or `pub` commands you will end up using the Flutter versions.
